packages/core/app/mirage/routes/report.js

```diff
@@ -3,22 +3,68 @@ import moment from 'moment';
 import RESPONSE_CODES from '../enums/response-codes';
 const TIMESTAMP_FORMAT = 'YYYY-MM-DD HH:mm:ss';
 
+/**
+ * @method getFilterParams
+ * @param {String} queryFilter
+ * @returns {Array} Filter parameters
+ * @description Parse filter parameters in the form of "(title==*H*,request==*Revenue*)";author==*ramvish*
+ * to a list of all the OR parameters, ie., [H, Revenue]
+ */
+const getFilterParams = function(queryFilter) {
```

Comment:
Would it be better to use a parsing library instead? https://github.com/vodori/rsql-parser=